create_source_tarball.py

```diff
@@ -0,0 +1,43 @@
+import os
+import tarfile
+
+WORKPATH = os.path.dirname(os.path.realpath(__file__))
+
+
+def read_yaml(stream):
+    # Alternative to yaml.safe_load
+    result = {}
+    current_section = None
+    for line in stream.readlines():
+        line = line.strip()
+        if line.strip().startswith("-"):
+            result.setdefault(current_section, []).append(line[1:].strip())
+        else:
+            current_section = line.strip(": ")
+            result.setdefault(current_section, [])
+    return result
```

Comment:
I am all right with this for now, but is not PyYAML fairly ubiquitous ? Is a `yaml` module not available on some of the machines you build on ? to me it would seem better to use a real `yaml` parser, but maybe you can explain more fully why you do not wish to do so.

Reply:
It was only not available on macos by default.
I avoided installing it to not increase the build times more.
But I'll just `pip install` it for simplicity.

Reply:
Ah, I see. I would like a real YAML parser better, myself, so that sounds good to me.

Reply:
The latest commits install it explicitly on all platforms.